board/safety/safety_subaru.h

```diff
@@ -256,7 +256,7 @@ static int subaru_tx_hook(CANPacket_t *to_send) {
     // reading ES button data by identifier (b'\x03\x22\x11\x30\x00\x00\x00\x00') is also allowed (DID 0x1130)
     bool is_button_rdbi = (GET_BYTES(to_send, 0, 4) == 0x30112203U) && (GET_BYTES(to_send, 4, 4) == 0x0U);
 
-    violation |= !(is_tester_present || is_button_rdbi);
+    violation |= !(is_tester_present || is_button_rdbi) || !subaru_longitudinal;
```

Comment:
for subaru, this is already checked by having a different set of allowed messages when using long